package.json

```diff
@@ -5,22 +5,24 @@
   "author": "Andrew Harmel-Law",
   "license": "CC-BY-NC-SA-4.0",
   "dependencies": {
-    "skill-scm-distributed-git-apprentice": "github:andrewharmellaw/skill-scm-distributed-git-apprentice",
-    "skill-networking-apprentice-dev": "github:andrewharmellaw/skill-networking-apprentice-dev",
-    "skill-unix-apprentice-user": "github:andrewharmellaw/skill-unix-apprentice-user",
-    "skill-ooad-apprentice-dev": "github:andrewharmellaw/skill-ooad-apprentice-dev",
+    "doc-publisher": "^0.1.15",
```

Comment:
Does it still all work if you put this line and the one below (pkginfo) down in devDependencies?  The idea was that we could keep the cognitive load light for most users - everything in "dependencies" would then be a skill.  It also kind of makes sense technically as the "runtime" dependencies _are_ the skills.

This then means we can _completely_(?) remove the publish_to_local_docs.js file, and call the whole thing from the travis build.

Result: no need to filter on "skill-*", less "code" in quests, simpler dependencies between the content (quests and skills) and the things which package / publish them.

Does that make sense? (It might not - you're the one doing this so let me know.)